hack/gh-workflow-ci.sh

```diff
@@ -125,10 +125,44 @@ run_e2e_tests() {
 }
 
 collect_logs() {
+  test_gitea_smee_url="${1}"
+  github_ghe_smee_url="${2}"
   mkdir -p /tmp/logs
   kind export logs /tmp/logs
   [[ -d /tmp/gosmee-replay ]] && cp -a /tmp/gosmee-replay /tmp/logs/
 
+  kind_dir=/tmp/logs/kind-control-plane
```

Comment:
what about just do a find -type ${kind_dir} |xargs -r sed 's/......//'

so we never going to miss any of them in the future....